IntelFsp2Pkg/Tools/SplitFspBin.py

```diff
@@ -12,6 +12,7 @@
 import struct
 import argparse
 from   ctypes import *
+from functools import reduce

```

Comment:
Do you still need this? I don't see it being used

Reply:
reduce is used by pre-existing, unmodified "Bytes2Val".  Python 3 requires this new import to call it.